drivers/spider.py

```diff
@@ -0,0 +1,114 @@
+"""
+  Spider3 Rider Driver
+"""
+
+import sys
+import os
+import inspect
+
+OSGAR_ROOT = os.path.realpath(os.path.abspath(os.path.join(os.path.split(inspect.getfile( inspect.currentframe() ))[0],"..")))
+if OSGAR_ROOT not in sys.path:
+    sys.path.insert(0, OSGAR_ROOT) # access to logger without installation
```

Comment:
What about this hack? Can we start requiring OSGAR_ROOT to be in PYTHONPATH?

Reply:
I would leave it for now, but remove it from all drivers is introduction of `osgar` package.

Reply:
ok